draft-ietf-cbor-packed.md

```diff
@@ -164,23 +169,34 @@ Argument reference:
 : A reference that combines a shared argument with a rump item as
   defined in {{sec-referencing-argument-items}}.
 
-Affix:
-: Prefix or suffix, used as an argument in an argument reference
-  employing the default function "concatenation".
+Rump:
+: The data item contained in an argument reference that is combined
+  with the argument to yield the reconstruction.
+
+Straight reference:
+: An argument reference that uses the argument as the left-hand side
+  and the rump as the right-hand side.
 
-Function reference:
-: An argument reference that uses a tag for argument, rump, or both,
-  causing the application of a function to reconstruct the data item.
+Inverted reference:
+: An argument reference that uses the rump as the left-hand side
+  and the argument as the right-hand side.
+
+Function tag:
+: A tag used in an argument reference for the argument (straight
+  references) or the rump (inverted references), causing the
+  application of a function indicated by the function tag to
+  reconstruct the data item.
```

Comment:
```suggestion
  application of a function indicated by the function tag during
  reconstruction of the data item.
```

Reply:
The old text (which might be embellished by "in order to") was a bit clearer that the reconstruction is the aim of the function application; and not just something that happens at the same time.

Reply:
That makes it indeed clearer, thanks!